### ISSUE TITLE
wrong comment style

### ISSUE THREAD

**Issue by thedolphin** [jvinet/knock#35]
Compiler generates warning

```
diff -ur knock-master/src/knockd.c knock-0.7.7/src/knockd.c
--- knock-master/src/knockd.c	2015-12-15 02:47:03.000000000 +0300
+++ knock-0.7.7/src/knockd.c	2015-12-17 15:06:25.434908265 +0300
@@ -19,9 +19,11 @@
  */

 #if __APPLE__
-// In MacOSX 10.5+, the daemon function is deprecated and will give a warning.
-// This nasty hack which is used by Apple themselves in mDNSResponder does
-// the trick.
+/*
+ * In MacOSX 10.5+, the daemon function is deprecated and will give a warning.
+ * This nasty hack which is used by Apple themselves in mDNSResponder does
+ * the trick.
+ */
 #define daemon deprecated_in_osx_10_5_and_up
 #endif
```

**Comment by airwoflgh** [comment]
Bad habit. Fixed in 0.7.7-fixes branch.

**Comment by airwoflgh** [comment]
Pull request back to master raised.
